packages/flutter_tools/templates/create/android.tmpl/buildSrc/src/main/groovy/FlutterPlugin.groovy

```diff
@@ -0,0 +1,229 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
```

Comment:
I have a couple of ideas I'll try out, to see if we can consume the plugin from the flutter root. I'll let this PR sit in the meantime.